platform/osx/display_server_osx.h

```diff
@@ -230,6 +230,7 @@ class DisplayServerOSX : public DisplayServer {
 	virtual Vector<int> get_window_list() const override;
 
 	virtual WindowID create_sub_window(WindowMode p_mode, uint32_t p_flags, const Rect2i &p_rect = Rect2i()) override;
+	virtual void show_window(WindowID p_id, uint32_t p_flags) override;
```

Comment:
You have not removed `p_flags` from macOS version.

Reply:
oooh, that's why! Very very sorry, i was convinced that the issue was still that line!

Reply:
Fixing it right now!